### ISSUE TITLE
About pretrained model

### ISSUE THREAD

**Issue by xxyll** [loiccordone/object-detection-with-spiking-neural-networks#14]
Hi, long time no see. 
I find a problem I hadn't noticed before. After training, there will be generate a folder ‘ckpt-od-gen1-vgg-11’, and there are three '.ckpt' files in it.
After training the mAp is 1.162, the loss of epoch49 is 0.744. But the files in ‘ckpt-od-gen1-vgg-11’ is 'gen1-epoch=00-train_loss=9.3256.ckpt', 'gen1-epoch=01-train_loss=2.2496.ckpt', 'gen1-epoch=02-train_loss=1.7713.ckpt'. It seems that only the results of the first three rounds have been saved.
Is the pretrained model used one of the three files in the test step？
